javaharness/java/arcs/api/PecPort.java

```diff
@@ -45,117 +48,107 @@
   private static final String HANDLE_REMOVE_MULTIPLE_MSG = "HandleRemoveMultiple";
   private static final String HANDLE_REMOVE_MSG = "HandleRemove";
 
-  private static final Logger logger = Logger.getLogger(PECInnerPortImpl.class.getName());
-
   private final String id;
-  private final ArcsEnvironment environment;
+  private final ShellApi shellApi;
   private final ParticleExecutionContext pec;
   private final ThingMapper mapper;
   private final PortableJsonParser jsonParser;
   private final IdGenerator idGenerator;
 
-  public PECInnerPortImpl(
-      String id,
-      String sessionId,
-      ArcsEnvironment environment,
-      ParticleExecutionContext pec,
-      PortableJsonParser jsonParser) {
+  public PecPort(
```

Comment:
I would prefer to keep the PECInnerPort name, as it is equivalent to the .ts implementation, see api-channel.ts (and we might be introducing PECOuterPort as well in post v0, when we switch to using the storage layer API)

Reply:
Renamed to PECInnerPort